src/main/java/core/basesyntax/model/converter/DataConverterImpl.java

```
@@ -0,0 +1,25 @@
+package core.basesyntax.model.converter;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputReport) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+
+        inputReport.stream()
+                .skip(1L)
+                .map(line -> line.split(","))
+                .filter(parts -> parts.length == 3)
```

you should throw an exception for invalid data